symphony-bdk-examples/bdk-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -1,10 +1,15 @@
 
 bdk:
-  host: acme.symphony.com
+  host: devx1.symphony.com
```

Comment:
Needs to be reverted, locally I prefer specifying those config params from IntelliJ (e.g. `--bdk.host=devx1.symphony.com`)